### ISSUE TITLE
Custom variables are added without any order

### ISSUE THREAD

**Issue by lukiq** [livechat/chat-window-ios#31]
Steps to reproduce:

Add to demo app: (for example Example-Swift app) this code:
```
// Setting some custom variables:
LiveChat.setVariable(withKey:"First variable name", value:"Some value")
LiveChat.setVariable(withKey:"Second name", value:"Other value")
LiveChat.setVariable(withKey:"3rd name", value:"Other value")
LiveChat.setVariable(withKey:"4th name", value:"Other value")
LiveChat.setVariable(withKey:"5th name", value:"Other value")
LiveChat.setVariable(withKey:"6th name", value:"Other value")
LiveChat.setVariable(withKey:"7th name", value:"Other value")
```

Result in the app: https://take.ms/NBjim

Expected result: 
Custom variables should be visible in the order that they were added in the code.

**Comment by kamil-szostakowski** [comment]
@lukiq  What should be an expected result of such a chain. 
```Swift
LiveChat.setVariable(withKey: "a", value: "1")
LiveChat.setVariable(withKey: "b", value: "2")
LiveChat.setVariable(withKey: "c", value: "3")
LiveChat.setVariable(withKey: "a", value: "4")
```
Assuming that the left most element is the first one.
```Swift
// a=4, b=2, c=3
// b=2, c=3, a=1
```
Which one is correct?

**Comment by alrosello** [comment]
I am not sure I understand the question.

But I would expect that the order in which we set the variables is the order they would show on the agent screen

A=1
B=1
C=3
A=4

**Comment by lukiq** [comment]
> @lukiq What should be an expected result of such a chain.
> 
> ```swift
> LiveChat.setVariable(withKey: "a", value: "1")
> LiveChat.setVariable(withKey: "b", value: "2")
> LiveChat.setVariable(withKey: "c", value: "3")
> LiveChat.setVariable(withKey: "a", value: "4")
> ```
> 
> Assuming that the left most element is the first one.
> 
> ```swift
> // a=4, b=2, c=3
> // b=2, c=3, a=1
> ```
> 
> Which one is correct?

This one should be correct.
```
// a=4, b=2, c=3
```

According to the documentation: https://developers.livechatinc.com/docs/js-api/#set-custom-variables, there is no option to sent variable with the same name.

**Comment by kamil-szostakowski** [comment]
Implemented in version 2.0.14.
